frc_characterization/logger_analyzer/templates/Robot.java.mako

```diff
@@ -152,10 +152,10 @@ public class Robot extends TimedRobot {
           % if not brushed:
       CANEncoder encoder = motor.getEncoder();
           % else:
-      CANEncoder encoder = motor.getEncoder(EncoderType.kQuadrature, ENCODER_EDGES_PER_REV);
+      CANEncoder encoder = motor.getEncoder(EncoderType.kQuadrature, ENCODER_EPR);
```

Comment:
Why change the name? It won't use the constant, right?

Reply:
ENCODER_EPR is just the integer type version of ENCODER_EDGES_PER_REV.  We could use the same variable and cast to type or something, honestly this seemed simplest